src/test/java/picard/sam/AbstractAlignmentMergerTest.java

```diff
@@ -104,6 +138,67 @@
                         default110LongR1Bases, default110LongR2Bases, sharedBases, sharedBases, default110LongR1ClippedBases, default110LongR2ClippedBases,
                         default110LongR1BaseQualities, default110LongR2BaseQualities, sharedQualities, sharedQualities, r1ClippedQualities10, r2ClippedQualities10},
 
+                /*
+                                SSSSSSSSSSSMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM->
+                                     <-MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMSSSSSSSSS
+
+                                soft clipping becomes:
+                                SSSSSSSSSSSMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMSS->
+                                     <-SSSSMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMSSSSSSSSS
+
+                                hard clipping leaves as from soft clipping
```

Comment:
huh?

Reply:
Oh…I had a hard time parsing that sentance...I'd say, `hard-clipping results in the same as with soft-clipping in this case.`

Reply:
the point is that when running with hard clipping turned on the result is the same as when running with only soft clipping.  Maybe "running with hard clipping gives same result as running with soft clipping" would be clearer?